packages/admin/admin-rte/src/core/translation/TranslationDialog.tsx

```diff
@@ -0,0 +1,54 @@
+import { messages } from "@comet/admin";
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, Grid } from "@mui/material";
+import { EditorState } from "draft-js";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import Rte from "../Rte";
+import RteReadOnly from "../RteReadOnly";
+
+interface TranslationDialogProps {
+    open: boolean;
+    close: () => void;
+    originalText: EditorState;
+    translatedText: EditorState;
+    editTranslation: (newValue: EditorState) => void;
+    applyTranslation: (newValue: EditorState) => void;
+}
+
+export const TranslationDialog: React.FC<TranslationDialogProps> = (props) => {
+    const { open, close, originalText, translatedText, editTranslation, applyTranslation } = props;
+
+    return (
+        <Dialog open={open} onClose={close} fullWidth maxWidth="lg">
+            <DialogTitle>
+                <FormattedMessage id="comet.translator.translation" defaultMessage="Translation" />
+            </DialogTitle>
+            <DialogContent>
+                <Grid container spacing={4} columns={2} alignItems="center">
+                    <Grid item xs={1}>
+                        <RteReadOnly value={originalText} />
+                    </Grid>
+                    <Grid item xs={1}>
+                        <Rte value={translatedText} onChange={editTranslation} />
+                    </Grid>
+                </Grid>
+            </DialogContent>
+            <DialogActions>
+                <Button
+                    onClick={() => {
+                        applyTranslation(translatedText);
+                        close();
+                    }}
+                    color="primary"
+                    variant="contained"
+                >
+                    <FormattedMessage {...messages.apply} />
+                </Button>
+                <Button onClick={close} color="primary">
+                    <FormattedMessage {...messages.cancel} />
+                </Button>
```

Comment:
IMO you got the order of buttons wrong: Cancel should be on the left side, Apply on the right. Could you check that with UX please?

Reply:
df7c1a116c7286931c9731969dac77e4de8fe285